tsconfig.strict.json

```diff
@@ -131,6 +131,8 @@
     "./lib/util/regex.ts",
     "./lib/util/sanitize.ts",
     "./lib/util/split.ts",
+    "./lib/versioning/swift/index.spec.ts",
+    "./lib/versioning/swift/index.ts",
```

Comment:
How about range?

Reply:
Good idea, I'll convert `files` to `include` once current PRs are done